src/assets/styles/overrides.scss

```diff
@@ -7,7 +7,8 @@
   text-transform: none;
   font-weight: 500;
   letter-spacing: normal;
-
+  text-transform: none !important; //otherwise the label will be uppercase
```

Comment:
Oh, so this replaces line 7?

Reply:
yeah... miss line 7, updated with adding "important"